dvc/config.py

```diff
@@ -52,6 +52,20 @@ def supported_loglevel(level):
     return level in ["info", "debug", "warning", "error"]
 
 
+def supported_hash_local(hash_names):
+    """Checks if hash config option has a valid value.
+
+    Args:
+        hash_names (list/string): hash name(s).
+    """
+    if isinstance(hash_names, str):
+        hash_names = [h.strip() for h in hash_names.split(",")]
```

Comment:
i think it would be beneficial to make it case-insensitive as well with `lower()`?